### PR TITLE
Add a Dockerfile based on ffig-org/ffig-base.

### PR DESCRIPTION
**Pull request by ajbennieston**
The image adds flask and the ffig source, then copies in the content of
this repository.

### CODE REVIEW

**Review by petr-tik — COMMENTED** [review]
ffig-org/ffig-base isn't a dockerhub repo

It is
https://hub.docker.com/r/ffig/

**Comment by ajbennieston** [comment]
ok, so it'll be ffig/ffig-base?

**Comment by petr-tik** [comment]
That's what the docker repo is called now, this dockerfile would break a `docker pull`

**Comment by ajbennieston** [comment]
What do you mean?

**Comment by petr-tik** [comment]
There is no ffig-org dockerhub repo.

**Comment by ajbennieston** [comment]
Well, yes. I knew that. What I didn't know was that there was one called `ffig`. I had worked on the basis that Jon would create one, possibly called `ffig-org`. It's changed now, so presumably this Dockerfile won't break anything?

(Except, of course, that the `ffig-base` image isn't published there yet!)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+FROM ffig/ffig-base
+MAINTAINER support@ffig.org
+
+# Install flask
+RUN pip install --upgrade pip && pip install flask
+RUN pip3 install --upgrade pip && pip install flask
+
+# Install the ffig codebase
+WORKDIR /home/ffig
+ADD https://github.com/FFIG/ffig-base/archive/master.zip /home/ffig/ffig
```
Comment (petr-tik) [inline Dockerfile]: https://github.com/FFIG/ffig-base

Will this not just copy the repo with the Dockerfile, which we used to build ffig/ffig-base?

Comment (petr-tik) [reply Dockerfile]: what about below?
`ADD https://github.com/FFIG/ffig/archive/master.zip /home/ffig/ffig`

**Review by petr-tik — COMMENTED** [review]
ok, no worries. I remember Jon sending me a message on slack about this (was probably a DM). 

Nothing to pull yet, I won't build it yet

File: Dockerfile (hunk 1 above)
Comment (ajbennieston) [reply Dockerfile]: Ah, sorry - yes that's what I meant to write :)

**Review by petr-tik — APPROVED** [review]

**Pull request merged**